walkers/src/tiles.rs

```diff
@@ -230,7 +230,11 @@ mod tests {
         let request = anticipated.expect().await;
         assert_eq!(
             request.headers().get(header::USER_AGENT),
-            Some(&HeaderValue::from_static("Walkers"))
+            Some(&HeaderValue::from_static(concat!(
+                env!("CARGO_PKG_NAME"),
```

Comment:
```suggestion
                "Walkers",
```

I prefer it to be explicit in this case. At least in tests.